uplink/decorators.py

```diff
@@ -286,12 +286,12 @@ def update_user(self, **info: Body):
     specify JSON fields separately, across multiple arguments:
 
     Example:
-    .. code-block:: python
+        .. code-block:: python
 
-        @json
-        @patch(/user")
-        def update_user(self, name: Field, email: Field("e-mail"):
-            \"""Update the current user.\"""
+            @json
+            @patch(/user")
+            def update_user(self, name: Field, email: Field("e-mail"):
```

Comment:
Whoops! Looks like I forgot to add a trailing parenthesis here 🤦‍♂️

Reply:
Don't worry, I missed it as well even though I was triple checking! Good catch, I've added that to the PR 👍